src/ImageSharp/ColorProfiles/CieLab.cs

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace SixLabors.ImageSharp.ColorProfiles;
+
+/// <summary>
+/// Represents a CIE L*a*b* 1976 color.
+/// <see href="https://en.wikipedia.org/wiki/Lab_color_space"/>
+/// </summary>
+public readonly struct CieLab : IProfileConnectingSpace<CieLab, CieXyz>
+{
+    /// <summary>
+    /// D50 standard illuminant.
+    /// Used when reference white is not specified explicitly.
+    /// </summary>
+    public static readonly CieXyz DefaultWhitePoint = KnownIlluminants.D50;
```

Comment:
See the other comment about property / field. I won't comment on these later on.

Reply:
I don't actually need this property. Was missed during the cleanup.